eqllib/analytics/collection/T1005-txt-files-with-command-line-redirection.toml

```
@@ -0,0 +1,23 @@
+[analytic.metadata]
+categories = ["detect"]
+confidence = "medium"
+contributors = ["Daniel Stepanic"]
+created_date = "12/04/2019"
+description = "Identifies Koadic's use of redirecting command output to plaintext files."
+id = "7c7f3114-7bdd-4477-a4e0-b5105b6babd8"
+name = "Creation of TXT File via ```cmd.exe``` with Command Line Redirection"
```

There are all RST, so it uses two ticks instead of three ` ``inline code`` `. I would strip it from the name regardless though, because that field is probably more picky.
```suggestion
name = "Creation of TXT File via cmd.exe with Command Line Redirection"
```

---

This is a cool idea. Since WMI can't give you stdout, the only to get output from commands executed locally or remotely via WMI is to reroute to a file. I think there are other groups that do this too. This could be a good one for a hunt, because I I've seen noise because administrators have done this too.

---

Thanks.  Yeh, it's an interesting technique and solid rule related to scripts/administrator activity.